utils/bazel/llvm-project-overlay/llvm/BUILD.bazel

```diff
@@ -1490,6 +1504,22 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "SandboxVectorizer",
+    srcs = glob([
+        "lib/Transforms/Vectorize/SandboxVectorizer/*.cpp",
```

Comment:
for consistency with the CMake build, this should be part of `Vectorize` above

Reply:
Done